### DISCUSSION TITLE
Accessing string value from a Swift GoTrue.AnyJSON

### DISCUSSION THREAD

**Discussion by mlawlerau** [supabase/supabase#13209] [Questions]
Hi, using the supabase swift client, when I access the User.usermetaData map, the entries are stored as GoTrue.AnyJSON.string() objects.

The .value member of GoTrue.AnyJSON is internal so I cant access it.

Whats the syntax for accessing the string value inside this object?

i.e. I can print it out ala:
userMetadata: ["username": GoTrue.AnyJSON.string("michael123")]

I know how to access the value object associated with "username" but I want to access the 'michael123' string and I dont know how to do that?

**Comment by peytonshetler** [comment]
@mlawlerau  I'm having the same problem access user metadata in my Swift/Supabase app... I guess there's no answer here? 

Would love to access the info directly from the metadata, instead of making a secondary network request to a Public table.
